src/client/ducks/repositories/selectors/index.js

```diff
@@ -72,3 +72,13 @@ export const getRepositoryId = createSelector(
     _.findKey(entities, (v) => (v.fullName === repoName))
 )
 
+export const getChangelogFetchStatus = statusFetchFactory(types.FETCH_CHANGELOG)
+export const getChangelog = (state: Object, props: Object) => {
+  const changesets = _.get(state, ['entities', 'changesets'], [])[undefined]
```

Comment:
yes, that's incorrect :) it's normalization issue, we should check ducks/schema/index.js, let me check it